docs/source/installation.rst

```diff
@@ -374,19 +390,29 @@ And build sktime from this environment:
 
     python setup.py install
 
-Replace ``x64`` by ``x86`` to build for 32-bit Python.
-
 Some users have experienced issues when installing NumPy, particularly version 1.19.4. Note that a recent Windows update may affect compilation using Visual Studio (see `Windows update issue <https://developercommunity.visualstudio.com/content/problem/1207405/fmod-after-an-update-to-windows-2004-is-causing-a.html>`_).
 
+Setting up a Conda environment
+""""""""""""""""""""""""""""""
 If you run into a problem installing the development version and are using Anaconda, try:
 
-1. Install Anaconda
-2. Create new environment: :code:`conda create -n sktime-dev python=3.8`
-3. Activate environment: :code:`conda activate sktime-dev`
-4. Install NumPy (pinned to 1.19.3) from pip: :code:`pip install numpy==1.19.3`
-5. Install requirements: :code:`pip install -r build_tools/requirements.txt`
-6. Follow the instructions above to point to "vcvarsall.bat"
-7. Run :code:`pip install --verbose --no-build-isolation --editable .`
+1. Open up an anaconda terminal
+2. Navigate to your local sktime folder :code:`cd sktime`
+3. Create new environment: :code:`conda create -n sktime-dev python=3.8`
+4. Activate environment: :code:`conda activate sktime-dev`
+5. Install required packages:
+    1. :code:`pip install numpy=1.19.3`
+    2. :code:`pip install cython`
+    3. :code:`pip install -r build_tools/requirements.txt`
+    4. If **fbprophet** fails to install:
+      1. :code:`conda install -c anaconda ephem`
+      2. :code:`conda install -c conda-forge install -c pystan`
+      3. :code:`conda install -c conda-forge install -c fbprophet`
+      4. Verify all requirements are satisfied by running :code:`pip install -r build_tools/requirements.txt` with no errors
```

Comment:
what to do if the requirements fail to verify? Try again from where, how often? Bug report on sktime? Ask for help on gitter?

Reply:
same below. Would be helpful if there was general advice, perhaps at the start, what to do if it doesn't work. I guess try again in a clean env, and if it doesn't work multiple times, ask for help?